ASV-BENCHMARKING.md

```diff
@@ -0,0 +1,82 @@
+Summarised below are the steps required to run the asv benchmark on a new computer.
+
+1. Clone trackintel and checkout to master. Add the trackintel root to python path 
+
+>(**For testing purposes only,** clone the abcnishant007/trackintel and checkout to `asv-trackintel` branch). 
+*@Ye, we need to edit this line after pull request is approved. *
+
+2. Download the larger geolife data from [this branch](https://github.com/abcnishant007/trackintel/tree/benchmark-files) and place it inside `tests/data/geolife_long_10_MB`. This dataset will not be used by default. The dataset option is hardcoded in the benchmark files as shown [here](https://github.com/abcnishant007/trackintel/blob/7b8c2ee2f12d98b59578cd0519aae6a5240ade4c/benchmarks/preprocessing_benchmarks.py#L6). The hardcoding exists temporarily only to ensure that we can switch to the smaller dataset for asv testing purposes on a new computer. Once the whole setup is complete, we can start using the bigger dataset. The hardcoding can be removed (or retained for future setups) and only the `geolife_long_10_MB` data set should be used for actual benchmarks.
```

Comment:
I am not sure I understand here. From the line of code, which dataset are we using now and which in the future?

Reply:
@hong2223 The stats which are live at the trackintel link are based on the big 10 MB data set. 
However, for testing purposes, I put an option to use the dataset which is readily available in the tests folder ( the geolife_long dataset) so that the results are obtained faster. I have updated the readMe with the following permalink:
https://github.com/abcnishant007/trackintel/blob/7b8c2ee2f12d98b59578cd0519aae6a5240ade4c/benchmarks/preprocessing_benchmarks.py#L6-L7 
where we assign the dataset name to the variable `bm_dataset`.